source/common/modules/markdown-editor/plugins/render-math.ts

```diff
@@ -15,32 +15,29 @@
 import { getBlockMathRE, getInlineMathRenderRE } from '@common/regular-expressions'
 import CodeMirror from 'codemirror'
 import katex from 'katex'
-
+import { IpcRenderer } from 'electron'
 import 'katex/contrib/mhchem' // modify katex module
+import { LatexCommand } from '@providers/latex-commands'
+
+const ipcRenderer: IpcRenderer = (window as any).ipc
```

Comment:
These calls have been properly typed in 2.2.0, so we can get rid of a lot of the fuzz!